server/service/Converter.py

```diff
@@ -0,0 +1,53 @@
+import json
+
+from server.service.data.Document import Cell, Row, Table, Document
+
+
+class Converter:
+
+    @staticmethod
+    def convert_to_json(document: Document):
+        result = {}
+        table_num = 1
+
+        for table in document.tables:
+            t = {}
+            row_num = 1
+            for row in table.rows:
+                r = []
+                for cell in row.cells:
+                    r.append(cell.text)
+                t["row" + str(row_num)] = r
+                row_num += 1
+            result["table" + str(table_num)] = t
+            table_num += 1
+        return json.dumps(result)
+
+
+# code for tests
+
```

Comment:
Testy potem usuniemy, ale póki co byłoby dobrze dodać ten if __name__ == ... by przy importowaniu to nie wykonywało się